The-SMIRNOFF-force-field-format.md

```diff
@@ -426,27 +454,32 @@ For convenience, and optional attribute specifies a torsion multiplicity by whic
 The default behavior of this attribute can be controlled by the top-level attribute `default_idivf` (default: `"auto"`) for `<ProperTorsions>`, which can be an integer (such as `"1"`) controlling the value of `idivf` if not specified or `"auto"` if the barrier height should be divided by the number of torsions impinging on the central bond.
 For example:
 ```XML
-<ProperTorsions potential="charmm" phase_unit="degrees" k_unit="kilocalories_per_mole" default_idivf="auto">
-   <Proper smirks="[a,A:1]-[#6X4:2]-[#6X4:3]-[a,A:4]" periodicity1="3" phase1="0.0" k1="1.40"/>
+<ProperTorsions version="0.3" potential="k*(1+cos(periodicity*theta-phase))" default_idivf="auto">
+   <Proper smirks="[a,A:1]-[#6X4:2]-[#6X4:3]-[a,A:4]" periodicity1="3" phase1="0.0*degree" k1="1.40*kilocalories_per_mole"/>
    ...
 </ProperTorsions>
 ```
 
-Currently, only `potential="charmm"` is supported, where we utilize the functional form:
+Currently, only `potential="k*(1+cos(periodicity*theta-phase))"` is supported, where we utilize the functional form:
 ```
 U = \sum_{i=1}^N k_i * (1 + cos(periodicity_i * phi - phase_i))
 ```
 
 .. note:: **AMBER defines a modified functional form**, such that `U = \sum_{i=1}^N (k_i/2) * (1 + cos(periodicity_i * phi - phase_i))`, so that barrier heights would need to be divided by two in order to be used in the SMIRNOFF format.
 
-If the `potential` attribute is omitted, it defaults to `charmm`.
+If the `potential` attribute is omitted, it defaults to `k*(1+cos(periodicity*theta-phase))`.
```

Comment:
Same here. If I understand `potential` can be omitted, but it's listed among the "required" attributes.